R/error-checks.R

```diff
@@ -71,14 +71,20 @@ isOfType <- function(object, type, nullAllowed = FALSE) {
 #' isIncluded(list("x", 1), list("a", "b", "x")) # FALSE
 #' @export
 isIncluded <- function(values, parentValues) {
+  if (is.vector(values)) {
```

Comment:
I don't mind dependencies but it seems to be overkill to do some basic R stuff no
I'd suggest 
```
values  <- c(values)
hasObject <- any(mapply(function(x) { is.object(x) },values  ))
if(hasObject)
```

less line and one dep less

Reply:
probably another version of x_apply is enough....lapply, sapply etc..

Reply:
I avoid using *apply family because it is type unstable, here since we know the type we expect, shouldn't cause any surprises.

I have dropped the `{purrr}` dependency now.

Reply:
What do you mean with "type unstable"? 
We are returning boolean so our mapped values are boolean right?